cms/envs/common.py

```diff
@@ -54,11 +54,13 @@
     # The following PROFILE_IMAGE_* settings are included as they are
     # indirectly accessed through the email opt-in API, which is
     # technically accessible through the CMS via legacy URLs.
-    PROFILE_IMAGE_BACKEND, PROFILE_IMAGE_DEFAULT_FILENAME, PROFILE_IMAGE_DEFAULT_FILE_EXTENSION,
```

Comment:
@aamishbaloch Was `PROFILE_IMAGE_DEFAULT_FILE_EXTENSION` removed from the list of imports by accident?

Reply:
@mtyaka Yes, it was skipped in changing the place of these settings. Created a PR [992](https://github.com/edx-solutions/edx-platform/pull/992) to add this.